deepchem/rl/__init__.py

```diff
@@ -19,12 +19,22 @@ class Environment(object):
   running in different processes or even on different computers.
   """
 
-  def __init__(self, state_shape, n_actions):
+  def __init__(self, state_shape, n_actions, state_dtype=None):
     """Subclasses should call the superclass constructor in addition to doing their own initialization."""
     self._state_shape = state_shape
     self._n_actions = n_actions
     self._state = None
     self._terminated = None
+    if state_dtype is None:
+      # Assume all arrays are float32.
+      import numpy
```

Comment:
To avoid polluting the namespace.  This is in `__init__.py`, so anything imported at the top level will become part of `dc.rl`.